android/browserstack_sample_local.py

```diff
@@ -24,33 +19,19 @@
 
     # Set other BrowserStack capabilities
     "bstack:options": {
-        "userName" : userName,
-        "accessKey" : accessKey,
-        "projectName" : "First Python Local project",
-        "buildName" : "browserstack-build-1",
-        "sessionName" : "BStack local_test",
-        "local" : "true"
+        "projectName" : "BrowserStack Samples",
```

Comment:
same bstack:options can be removed, same for ios files

Reply:
done